ambry-router/src/main/java/com.github.ambry.router/PutManager.java

```diff
@@ -245,9 +247,14 @@ void onComplete(PutOperation op) {
     if (e != null) {
       blobId = null;
       routerMetrics.onPutBlobError(e);
-      routerCallback.scheduleDeletes(op.getSuccessfullyPutChunkIdsIfComposite());
+      routerCallback.scheduleDeletes(op.getSuccessfullyPutChunkIdsIfComposite(), op.getServiceId());
```

Comment:
As mentioned above, this is actually just the service ID suffix, which will be appended to "ambry-background-delete-". I will add something to differentiate it from regular deletes, though. Maybe "ambry-background-delete-failed-put-serviceid"